src/eko/output/struct.py

```diff
@@ -750,16 +750,18 @@ def detached(cls, theory: dict, operator: dict, path: pathlib.Path):
         """
         bases = operator["rotations"]
         for basis in ("inputgrid", "targetgrid", "inputpids", "targetpids"):
-            bases[f"_{basis}"] = bases[basis]
-            del bases[basis]
+            if basis in bases:
+                bases[f"_{basis}"] = bases[basis]
+                del bases[basis]
         bases["pids"] = np.array(br.flavor_basis_pids)
         for k in ("xgrid", "_inputgrid", "_targetgrid"):
```

Comment:
Dropping the former loop, also this loop has to be dropped

Reply:
This is what I did before but then the tests are really a problem. Of course fixing the tests would be the correct solution but, since at the moment we are not using at all `rotations`, I believe that in the next future we need to get rid completely of it. This will be the proper change and at that moment we can drop these loops and fix the tests accordingly. 

BTW I am saying that we are not using at all `rotations` because even if I use `x_grid_reshape` and it changes `targetgrid`, then `load()` would completely ignore `targetgrid` and would set it to `xgrid` (wrongly because instead the operator is rotated). This is the problem that I have with pineko. So now `rotations` is not even keeping track of the rotations. It is completely useless.

Reply:
> BTW I am saying that we are not using at all `rotations` because even if I use `x_grid_reshape` and it changes `targetgrid`, then `load()` would completely ignore `targetgrid` and would set it to `xgrid`

This should not happen: we want to "ignore" `targetgrid` (i.e. not putting it at all) only in the runcard. The reason why it is failing in Pineko is different: metadata are not automatically updated, and you are not doing it specifically.
Metadata are one thing, and runcards are another. Incidentally, they both have `rotations` - but we can drop `rotations` at all from the runcard, since we'll never use it any longer.